tools/static-code-analysis/checkstyle/suppressions.xml

```diff
@@ -16,4 +16,6 @@
     <!-- Eclipse SmartHome specific suppressions-->
     <!-- These bundles are generated trough XText -->
     <suppress files=".+org.eclipse.smarthome.model.+" checks="RequireBundleCheck|ExportInternalPackageCheck|ManifestPackageVersionCheck|ImportExportedPackagesCheck"/>
+    <!-- Some source files have different headers -->
+    <suppress files=".+org.eclipse.smarthome.automation.+" checks="HeaderCheck"/>
```

Comment:
Yes, because we currently have a different arrangement for the license header of the automation component.